_episodes/00-geohack-intro.md

```diff
@@ -21,59 +21,17 @@ This tutorial is a preliminary tutorial that we would like you to complete befor
 
 ### Overview
 
-Python software is distributed as a series of _libraries_ that are called within your code to perform certain tasks. There are many different collections, or _distributions_ of Python software. Generally you install a specific distribution of Python and then add additional libraries as you need them. There are also several different _versions_ of Python. The two main versions right now are 2.7 and 3.5. Some libraries only work with specific versions of Python.
+Python software is distributed as a series of *libraries* that are called within your code to perform certain tasks. There are many different collections, or *distributions* of Python software. Generally you install a specific distribution of Python and then add additional libraries as you need them. There are also several different *versions* of Python. The two main versions right now are 2.7 and 3.6. Some libraries only work with specific versions of Python.
 
 So even though Python is one of the most adaptable, easy-to-use software systems, you can see there are still complexities to work out and potential challenges when delivering content to a large group. Therefore we have a number of different ways that we are trying to simplify this process to maximize your learning during Geohackweek.
 
-Our recommended method is to use [docker](https://www.docker.com/), a system for _containerizing_ computer platforms. We are distributing our tutorial content as a series of these containers. Once installed, you can run Python within these virtual environments on any platform. Even if you are already an experienced Python user, we recommend you using docker. It is a very powerful method for sharing software implementations, and is something worth learning. The challenge with using docker is that it is relatively new and is optimized for specific, modern versions of software and hardware. Those of you with older computers or software versions may encounter challenges in using docker effectively. 
+This year, we will be using [JupyterHub](https://jupyterhub.readthedocs.io/en/latest/) on [Amazon Web Services (AWS) Cloud](https://aws.amazon.com/). JupyterHub is a multi-user Hub that provides multiple instances of the single-user Jupyter notebook server. A Jupyter Notebook is an open-source web application that allows users to create and share documents containing live code, equations, visualizations, and markdown texts.
```

Comment:
This year, we will be using [JupyterHub](https://jupyterhub.readthedocs.io/en/latest/) hosted on [Amazon Web Services (AWS) Cloud](https://aws.amazon.com/).